packages/flutter_tools/lib/src/base/io.dart

```diff
@@ -0,0 +1,63 @@
+import 'dart:io' as io show exit, exitCode;
+
+import 'package:meta/meta.dart';
+
+export 'dart:io'
+    show
+        BytesBuilder,
+        exitCode,
+        GZIP,
+        InternetAddress,
+        IOException,
+        IOSink,
+        HttpClient,
+        HttpClientRequest,
+        HttpClientResponse,
+        HttpHeaders,
+        HttpRequest,
+        HttpServer,
+        HttpStatus,
+        pid,
+        Platform,
+        Process,
+        ProcessException,
+        ProcessResult,
+        ProcessSignal,
+        ProcessStartMode,
+        ServerSocket,
+        stderr,
+        stdin,
+        stdout,
+        Socket,
+        SocketException,
+        SYSTEM_ENCODING,
+        WebSocket,
+        WebSocketTransformer;
```

Comment:
I'd add a comment explaining that we shouldn't add File IO related function because folks should use package:file instead.